docusaurus.config.js

```diff
@@ -130,6 +132,20 @@ const config = {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
       },
+      algolia: {
```

Comment:
Nice, :1st_place_medal:

Reply:
Yes, but currently, it is not functional. It will be available after hosting.